style.css

```diff
@@ -0,0 +1,106 @@
+
+body {
+  font-family: Tahoma, Geneva, sans-serif;
+}
+
+.main {
+  background-image: url(https://dl.airtable.com/.attachmentThumbnails/ad97dd00af8cdd4a666f5cf5a0d56bcc/451c6cae);
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  overflow-y: scroll;
+  font-size: 24px;
+  color: #fff;
+}
```

Comment:
I think it would be better to use flexbox because instead of float or position .The Flexible Box Layout Module  makes it easier to design flexible responsive layout structure without using float or positioning. And also A flex container expands items to fill available free space or shrinks them to prevent overflow.